### DISCUSSION TITLE
Getting Unable to find a required configuration file at '/config//runtime.yaml' error

### DISCUSSION THREAD

**Discussion by SumeetWarankar** [tomkerkhove/promitor#1761] [Q&A]
Hello,

I have created a docker-compose file in which I am using "ghcr.io/tomkerkhove/promitor-agent-scraper:2.5.0" docker image. So, after running docker-compose file I am getting below error.
`[15:43:57 FTL] Unable to find a required configuration file at '/config//runtime.yaml'`

Could you please help me to resolve this issue?

**Comment by tomkerkhove** [comment]
Are you mounting the configuration to that location?

This is the Docker Compose configuration that I use during development: https://github.com/tomkerkhove/promitor/blob/master/src/docker-compose.yml

**Reply by SumeetWarankar** [reply]
Hi Tom,
Below is the location where I am mounting.
`volumes:
      - ./scraper/metrics-declaration.yaml:/config/metrics-declaration.yaml
      - ./scraper/runtime.yaml:/config/runtime.yaml`

**Reply by tomkerkhove** [reply]
Do the file locations exist and do they successfully get added? For example, can you `cat /config/runtime.yaml` inside the container?

Can you add the full logs please?
